packages/react-cosmos-router-proxy/src/index.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { string } from 'prop-types';
+import { MemoryRouter, Route } from 'react-router';
+import proxyPropTypes from 'react-cosmos-utils/lib/proxy-prop-types';
+import LocationInterceptor from './LocationInterceptor';
+
+export default () => {
+  const RouterProxy = props => {
+    const { nextProxy, fixture, onFixtureUpdate } = props;
+    const { value: NextProxy, next } = nextProxy;
+    const { route, url } = fixture;
+
+    const children = (
+      <LocationInterceptor onLocation={url => onFixtureUpdate({ url })}>
```

Comment:
`onFixtureUpdate` updates the original fixture and sends it to the Playground, which updates the fixture editor. So if you look into the gif I posted, you'll notice that the route updates in the fixture editor as you click on links inside the component. Does that answer your question?

PS. The actual postMessage event is `fixtureUpdate`, described [here](https://github.com/react-cosmos/react-cosmos/blob/master/docs/playground-loader.md#fixture-updates).